pym/bob/generators/QtCreatorGenerator.py

```diff
@@ -194,7 +205,14 @@ def generateQtProject(package, destination, updateOnly, projectName, args):
         buildMe = []
         buildMe.append("#!/bin/sh")
         buildMe.append("bob dev $1 " + args + " " + project )
-        buildMe.append("bob project -n qt-creator " + project + " -u --destination " + destination + ' --name ' + projectName)
+        projectCmd = "bob project -n qt-creator " + project + " -u --destination " + destination + ' --name ' + projectName
+        for i in includeDirs:
+            projectCmd += " -I " + i
+        if filter:
+            projectCmd += " --filter " + filter
```

Comment:
Same here. Regex strings are especially sensitive and must be quoted correctly...